packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -31,12 +31,17 @@ const enableKernelPromiseGC = true;
 // The schema is:
 //
 // vat.names = JSON([names..])
+// vat.dynamicIDs = JSON([vatIDs..])
```

Comment:
Let me know when you think the schema changes have stabilized, as I (or somebody, but probably me) will need to update the `kerneldump` utility to match.

Reply:
It's probably stable enough now. Unless it breaks something (which, if there were unit tests for `kernelDump`, I would have noticed :wink:), I think we can ignore the new keys until your dead-vat changes land, and update `kernelDump` for both at the same time.

Reply:
Adding to the database schema doesn't break `kernelDump` but if it knows about the new items it can display them more usefully.